common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -863,3 +869,77 @@ def get_transcripts_info(self, is_bumper=False, include_val_transcripts=False):
             "sub": sub,
             "transcripts": transcripts,
         }
+
+
+def get_transcript_from_content_store(video_descriptor, lang=None, feature_enabled=False):
+    """
+    Get video transcript from content store.
+
+    Arguments:
+        video_descriptor (Video Descriptor): course identifier
+        lang (unicode): transcript language
+        feature_enabled (bool): whether to include transcript info from val
+
+    Returns:
+        tuple containing content, filename, mimetype
+    """
+    try:
+        transcripts = video_descriptor.get_transcripts_info(include_val_transcripts=feature_enabled)
+        content, filename, mimetype = video_descriptor.get_transcript(transcripts, lang=lang)
+    except (KeyError, ValueError):
+        raise NotFoundError(
+            u"Transcript not found for {}, lang: {}".format(video_descriptor.location.block_id, lang)
+        )
+
+    return content, filename.encode('utf-8'), mimetype
+
+
+def get_transcript_from_val(edx_video_id, lang=None, output_format=Transcript.SRT, feature_enabled=False):
+    """
+    Get video transcript from edx-val.
+
+    Arguments:
+        edx_video_id (unicode): course identifier
+        lang (unicode): transcript language
+        output_format (unicode): transcript output format
+        feature_enabled (bool): whether to include transcript info from val
+
+    Returns:
+        tuple containing content, filename, mimetype
+    """
+    transcript = None
+    if feature_enabled:
+        transcript = get_video_transcript_content(edx_video_id, lang)
+
+    if not transcript:
+        raise NotFoundError(u'Transcript not found for {}, lang: {}'.format(edx_video_id, lang))
+
+    transcript_conversion_props = dict(transcript, output_format=output_format)
+    transcript = convert_video_transcript(**transcript_conversion_props)
+    filename = transcript['filename']
+    content = transcript['content']
+    mimetype = Transcript.mime_types[output_format]
+
+    return content, filename.encode('utf-8'), mimetype
+
+
+def get_transcript(course_id, block_id, lang=None):
```

Comment:
we should also have a `transcript_format` parameter in order for us to be able to get transcript in specified format.